src/actions.js

```diff
@@ -0,0 +1,30 @@
+import axios from 'axios';
+
+export const NEXT_QUOTE = 'NEXT_QUOTE';
+export const nextQuote = () => ({
+    type: NEXT_QUOTE,
+});
+
+const getQuotes = () => axios.get('https://gist.githubusercontent.com/camperbot/5a022b72e96c4c9585c32bf6a75f62d9/raw/e3c6895ce42069f0ee7e991229064f167fe8ccdc/quotes.json');
+
+export const LOAD_QUOTES = 'LOAD_QUOTES';
+export const loadQuotes = () => dispatch => {
+    dispatch({
+        type: LOAD_QUOTES,
+    });
```

Comment:
Would be good to add a "payload" (or some other name) object inside action for consistency where quotes, error or any other data coming with action is stored.

Maybe type, payload, error would be also some convention, it could change the way we look at action in current project, we could put actions into 2 parts then requesting action and response action rather then request action, success action, failed action, but then a lot other things would change